core/arch/arm/pta/sdp_pa.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2017, ARM Limited, All Rights Reserved
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <kernel/pseudo_ta.h>
+#include <mm/core_memprot.h>
+#include <mm/tee_mmu.h>
+#include <sdp_pa.h>
+#include <string.h>
+
+#define PTA_NAME "sdp_pa.pta"
```

Comment:
`sdp_pa` as pseudoTA seems quite restrictive. If we were to add SDP services from the core, we could surely need a "SDP pTA" and likely rename current sources. Could you rename them now before an initial merge, to move from `sdp_pa` to `sdp_pta` ?  
**pta/sdp_pta.c** **lib/libutee/include/sdp_pta.h**  `#define PTA_NAME "sdp.pta"`

Reply:
Our original idea is only to support "pa" which mean physical address. Of course I can rename to sdp.pta to let it as a sdp services.